qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py

```diff
@@ -404,6 +404,15 @@ def simplify(self, atol=None, rtol=None):
             x = self.paulis.x[non_zero_indexes]
             z = self.paulis.z[non_zero_indexes]
             coeffs = coeffs[non_zero]
+
+            # Remove real (resp. imaginary) parts that are close to 0. This part for instance
```

Comment:
Just a question. Can atol and rtol same with total atol and rtol? Is there anything we want to set individually? (But if we allow individual setting, the API will become too complex, so I have no idea...)

Reply:
You mean individually for imaginary and real? Yeah that might become a bit more complex... but this could be a feature if we find a use-case. Without a use-case in mind I wouldn't add it though 🙂

Reply:
Yes. and also for https://github.com/Qiskit/qiskit-terra/blob/56c483cbe71236bd80110d531965235969b31895/qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py#L394.
I'm not sure we can share atol and rtol.

If no one reply to this thread, please go ahead.

Reply:
I thought about it for a night.

In other words, I am concerned about the point that two `isclose`s [1](https://github.com/Qiskit/qiskit-terra/blob/56c483cbe71236bd80110d531965235969b31895/qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py#L394) and [2](https://github.com/Qiskit/qiskit-terra/blob/56c483cbe71236bd80110d531965235969b31895/qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py#L412) use the same atol as the threshold even though they have different meanings.

Also, if you are going to chop real and imaginary parts, you should do the same for other Operators. However, we don't want to do that.

So I suggest we add another method like `SparsePauliOp.hermite_if_close(atol: Optional[float])`.

Reply:
I think Ikko and I are saying the exact same stuff but I didn't see it, so started my own comment chain, haha.

Reply:
Yes I think it's the same concern! See the comment below for the new method I added now 🙂